docs/tools-and-sdks.mdx

```diff
@@ -19,6 +19,10 @@ Allows other Stellar apps to request transaction signing or identity verificatio
 
 View all anchors on Stellar, their currencies, and where they operate.
 
+### [Asset Sandbox](https://stellar.cheesecakelabs.com/?utm_source=stellar&utm_medium=landing-page&utm_campaign=stellar-asset-sandbox)
+
+A sandbox supported by SDF and Cheesecake Labs for businesses to experiment with asset issuance on the Stellar's test network.
```

Comment:
```suggestion
A sandbox supported by SDF and Cheesecake Labs for businesses to experiment with asset issuance on Stellar's test network.
```

nitpick: `the` or `'s` feel... "conflicting" ?? not sure how to explain it but it reads to me like there should be one or the other, but not both

Also, would we want to link to Cheesecake Labs, too? i.e., `... SDF and [Chesecake Labs](https://cheesecakelabs.com/) for ...`